app/controllers/katello/api/v2/host_collections_controller.rb

```diff
@@ -149,6 +155,89 @@ def copy
       respond_for_create :resource => new_host_collection
     end
 
+    api :PUT, "/host_collections/:id/autoattach_subscriptions", N_("List summary of subscriptions on all content hosts in collection")
+    param :id, :identifier, :desc => N_("ID of the host collection"), :required => true
+    def autoattach_subscriptions
+      systems = Katello::System.readable
```

Comment:
I will switch it to System.editable. At the same time I'll open an RFE to allow RBAC content hosts where host collection is named (currently there is no way to set a host collection's content hosts editable).